Triphouse-project/index.html

```diff
@@ -5,11 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```

Comment:
![2021-05-02_21h07_52](https://user-images.githubusercontent.com/19935417/116822986-7ca5a880-ab8a-11eb-814e-227e23dd0017.png)
Good solution, keep up the good work.

Reply:
thank you